nixos/modules/services/backup/borgbackup.nix

```diff
@@ -226,6 +226,7 @@ let
 
 in {
   meta.maintainers = with maintainers; [ dotlambda ];
+  # Don't edit the docbook xml directly, edit the md and generate it using md-to-db.sh
   meta.doc = ./borgbackup.xml;
```

Comment:
Can't we just set `meta.doc = ./borgbackup.md;` and fix up the extension automatically?

Reply:
As an alternative, could we instead have that comment at the top of the generated XML files?